pkg/notifier/telegramnotifier/telegram.go

```diff
@@ -201,8 +211,15 @@ func (n *Notifier) NotifyTo(channel string, obj interface{}, args ...interface{}
 	}
 }
 
-func (n *Notifier) SendPhoto(buffer *bytes.Buffer) {
-	n.SendPhotoTo("", buffer)
+func (n *Notifier) Upload(file *types.UploadFile) {
+	select {
+	case n.taskC <- notifyTask{
+		photoBuffer: file.Data,
+	}:
+	case <-time.After(1 * time.Second):
```

Comment:
If `n.taskC` is full for 1 second, the `notifyTask` will be dropped without any logging message. It may be very difficult to debug this case if n.taskC is full